### ISSUE TITLE
bug: Capacitor run do not copy the source maps

### ISSUE THREAD

**Issue by distante** [ionic-team/capacitor#5962]
### Capacitor Version

```
Capacitor Doctor  💊 

Latest Dependencies:

  @capacitor/cli: 4.3.0
  @capacitor/core: 4.3.0
  @capacitor/android: 4.3.0
  @capacitor/ios: 4.3.0

Installed Dependencies:

  @capacitor/ios: not installed
  @capacitor/cli: 4.3.0
  @capacitor/core: 4.3.0
  @capacitor/android: 4.3.0

[success] Android looking great! 👌
```

### Platform(s)
Android

### Current Behavior
`cap run` and `cap run --inline`  does not copy the source maps

### Expected Behavior
The source maps should be copied when running `run --inline`

`npm --version` output: 8.11.0

`node --version` output: v16.15.1

### Additional Context
While deploying device-specific code, the source maps are needed.

**Comment by jcesarmobile** [comment]
there are no `--livereload` nor `livereload` params in capacitor run command, and also the source maps problem was not present when using live reload as the problem comes from copying the compiled files from one folder to another, which is not the case when using live reload since it uses the server url

Can you provide a sample app and steps to reproduce what you are doing?

**Comment by distante** [comment]
Sorry I went under the impression that still exists. Since I am using an `nx` mono repo I had to create my own script that puts the server URL into the capacitor.json.

But I will describe here the steps, They are really easy.

1. I Manually set the `server.url` into capacitor json.
2. I start my angular project `nx serve my-project --host=0.0.0.0 --port=${liveReloadPort}`
3. I call `run android --inline` (`nx run my-project:cap --cmd 'run android  --inline'`)
